docs/fastdds/discovery/general_disc_settings.rst

```diff
@@ -55,6 +55,10 @@ The possible values are:
 |                     +---------------------+--------------------------------------------------------------------------+
 |                     | |CLIENT|            | The DomainParticipant acts as a client for discovery traffic. |br|       |
 |                     |                     | It send its discovery information to the server, and receives |br|       |
```

Comment:
```suggestion
|                     |                     | It sends its discovery information to the server, and receives |br|       |
```

I am not sure if after the `and` which has a comma just before breaking the sentence, a new `it` is needed